app.js

```diff
@@ -59,3 +59,5 @@ const serverHttp = http
 
 // socket.io 서버 시작 및 app 인스턴스에 저장
 app.set("io", startSocketServer(serverHttp));
+
+require("./src/schedules/index")(app.get("io"));
```

Comment:
지금 이부분에서 
```
(node:91163) UnhandledPromiseRejectionWarning: ReferenceError: io is not defined
    at /Users/ptptsw/Workspace/taxi-back/src/schedules/notifyBeforeDepart.js:44:25
    at Array.map (<anonymous>)
    at Task.sendReminder [as _execution] (/Users/ptptsw/Workspace/taxi-back/src/schedules/notifyBeforeDepart.js:43:11)
```
에러가 발생하고 있는데, 구글링 해보니까 socket.io instance가 하니라 module handler 형식으로 불러와져서 생기는 문제같아요!
혹시 해결방법을 알고계실까요??

Reply:
```suggestion
const io = startSocketServer(serverHttp);
app.set ("io", io);

require ("./src/schedules")(io);
```

왠지 이렇게 코드 바꾸면 해결되지 않을까요(??)

Reply:
@ptptsw

Reply:
같은 에러 발생해서 좀 더 고민해보겠습니다!!

Reply:
startSocketServer를 promisify 해야할 까요??

Reply:
어떤 구조인지 감이 안잡히는데 코드로 보여주시죠 ㅋㅋㅋ